File: Assets/ToLua/Lua/user/ui/game/model/const_data.lua
local t = {}

local function Start()
	local origin = dofile('const')
	t.skill_hurt_a = tonumber(origin.t['skill_hurt_a'][2])
	t.float_hurt_add = tonumber(origin.t['float_hurt_add'][2])
	t.caron_hurt_add = tonumber(origin.t['caron_hurt_add'][2]) 
	t.caron_hurt_max = tonumber(origin.t['caron_hurt_max'][2])
	t.caron_hurt_time = tonumber(origin.t['caron_hurt_time'][2])
	t.ai_skill_round = tonumber(origin.t['ai_skill_round'][2])
end

Start()
return t

File: Assets/Code/Logic/Const/Model/ConstData.cs
using UnityEngine;
using System.Collections;
using Common.Util;
namespace Logic.Const.Model
{
    public class ConstData
    {
        private static ConstData _constData;

        public static ConstData GetConstData()
        {
            if (_constData == null)
            {
                _constData = CSVUtil.ParseClass<ConstData>("config/csv/const");
            }
            return _constData;
        }

        #region field
        [CSVElement("skill_hurt_a")]
        public float skillHurtA;
        
        [CSVElement("float_hurt_add")]
        public float floatHurtAdd;
        
        [CSVElement("caron_hurt_add")]
        public float comboHurtAdd;

        [CSVElement("caron_hurt_max")]
        public float comboHurtMax;

        [CSVElement("caron_hurt_time")]
        public float comboHurtDelay;

        [CSVElement("ai_skill_round")]
        public uint aiSkillRound;
        #endregion
    }
}